responses/01_make-a-plan.md

```diff
@@ -75,6 +75,9 @@ Mapping users, keeping commit comments, and other data migrations are more compl
 
 Choose the drop-down below that best fits your current situation or for a printable version of the steps in this course, check out the [Quick Reference Guide]({{ host }}/public/{{ course.slug }}.pdf).
```

Comment:
I don't think we recreated the quick reference guide when this course changed, @carolynshin. I think we need to remove the link.